### PR TITLE
[CTOR-511] Update Sensormetrix doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md [hunk 1]
```diff
@@ -2,42 +2,520 @@
...
+| Métrique    | Unité |
+|:------------|:------|
+| voltage     | V     |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md:103]: no prerequisites!

File: pp/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md [hunk 2]
```diff
@@ -2,42 +2,519 @@
...
+|:------------|:------|
+| voltage     | V     |
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md:102]: no prerequisites!

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md [hunk 3]
```diff
@@ -2,42 +2,518 @@
...
+	--hostname=10.0.0.1 \
+	--urlpath='/index.htm?ev'  \
+	--warning='' \
+	--critical=''
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK:  | 'voltage'=54V;;;;
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md:303]:
```suggestion
OK: Voltage: 54V | 'voltage'=54V;;;;
```

File: pp/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md [hunk 4]
```diff
@@ -2,42 +2,517 @@
...
+	--hostname=10.0.0.1 \
+	--urlpath='/index.htm?ev'  \
+	--warning='' \
+	--critical=''
+```
+
+The expected command output is shown below:
+
+```bash
+OK:  | 'voltage'=54V;;;;
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-sensors-sensormetrix-em01-web.md:302]:
```suggestion
OK: Voltage: 54V | 'voltage'=54V;;;;
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**